Counterexamples/CliffordAlgebra_not_injective.lean

```diff
@@ -0,0 +1,273 @@
+/-
+Copyright (c) 2021 Eric Wieser. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Eric Wieser
+-/
+import Mathlib.Algebra.CharP.Algebra
+import Mathlib.Algebra.CharP.Pi
+import Mathlib.Algebra.CharP.Quotient
+import Mathlib.Algebra.CharP.Two
+import Mathlib.Data.Matrix.Notation
+import Mathlib.Data.MvPolynomial.CommRing
+import Mathlib.Data.Nat.Prime
+import Mathlib.Data.ZMod.Basic
+import Mathlib.FieldTheory.MvPolynomial
+import Mathlib.LinearAlgebra.CliffordAlgebra.Basic
+import Mathlib.LinearAlgebra.Finsupp
+import Mathlib.RingTheory.MvPolynomial.Basic
+import Mathlib.RingTheory.MvPolynomial.Ideal
+
+/-! # `algebraMap R (CliffordAlgebra Q)` is not always injective.
+
+A formalization of https://mathoverflow.net/questions/60596/clifford-pbw-theorem-for-quadratic-form/87958#87958
```

Comment:
Can you add a short explanation of the example?

Reply:
Done